### ISSUE TITLE
other PG wearing inventory list

### ISSUE THREAD

**Issue by AleCillo** [arkayenro/arkinventory#1300]
Hi, a question, it's possible to assign a space to view, for example, the cloak of all my PG listed by PG name? This is for tracking Ashjra'kamas progress over alts.

Tyvm

Ale

**Comment by arkayenro** [comment]
no, sorry.  if you have bonus ids enabled (or youll just see the base item) you could just open the wearing window and swap to that character and check their cloak
